frontend/src/Admin/components/ShowSubmissionDetails.js

```diff
@@ -0,0 +1,184 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Row, Col } from 'reactstrap'
+
+import { getImageThumbnail } from '../../utils'
+
+// Submission types
+const VIDEO = 'VIDEO'
+const PHOTO = 'PHOTO'
+const OTHER = 'OTHER'
+
+const YOUTUBE_BASE_URL = 'https://www.youtube.com/watch?v='
+const VIMEO_BASE_URL = 'https://vimeo.com/'
+
+const PhotoThumbnail = styled.img`
+  height: auto;
+  max-height: 5em;
+  max-width: 100%;
+  min-width: 3em;
+  width: auto;
+`
+
+const renderSubmissionByType = submission => {
+  switch (submission.entryType) {
+    case PHOTO:
+      return (
+        <Fragment>
+          <PhotoThumbnail
+            alt={submission.title}
+            // TODO replace this with deployed URL
+            src={`//localhost:3000/static/uploads/${getImageThumbnail(
+              submission.path
+            )}`}
+          />
+          <dt>Dimensions</dt>
+          <dd>
+            {submission.horizDimInch} in. × {submission.vertDimInch} in.
+          </dd>
+          <dt>Media Type</dt>
+          <dd>{submission.mediaType}</dd>
+        </Fragment>
+      )
+    case VIDEO:
+      if (submission.provider === 'youtube') {
+        return (
+          <Fragment>
+            <dt>YouTube Video</dt>
+            <dd>
+              <a
+                href={`${YOUTUBE_BASE_URL}${submission.videoId}`}
+                target='_blank'
+              >
+                {`${YOUTUBE_BASE_URL}${submission.videoId}`}
+              </a>
+            </dd>
+          </Fragment>
+        )
+      } else {
+        return (
+          <Fragment>
+            <dt>Vimeo Video</dt>
+            <dd>
+              <a
+                href={`${VIMEO_BASE_URL}${submission.videoId}`}
+                target='_blank'
+              >
+                {`${VIMEO_BASE_URL}${submission.videoId}`}
+              </a>
+            </dd>
+          </Fragment>
+        )
+      }
+    case OTHER:
+      return (
+        <Fragment>
+          <dt>Other</dt>
+          <dd>
+            <a
```

Comment:
Check if the path ends with `.jpg`. If it does, render the image like we render photo submissions above.

If it's a PDF (aka, not a JPG), keep it how it is now, but change the link text to be "View PDF"